### DISCUSSION TITLE
time series: error in model loading and prediction model. AttributeError

### DISCUSSION THREAD

**Discussion by acartro** [pycaret/pycaret#2861] [Q&A]
Good morning,

For a timeseries project, I have made a randomforest, lightgbm, linear regression, with exogenous variables and without exogenous variables. 
I see that when I load the model that I have stored and I use the predict_model function, I get an error 'ConditionalDeseasonalizer' object has no attribute '_X'.

Why does this happen? How can I use my model to predict?

![image](https://user-images.githubusercontent.com/11137446/185072403-12390f80-ad10-4b5d-94f6-dc3cc5d6f709.png)

**Comment by celestinoxp** [comment]
Hi,
can you provide the versions of the software you are using?
Please paste the output of `from pycaret import show_versions; show_versions()`. If this does not work, try this instead `import pycaret; pycaret.__version__` and version of all software involved.

**Reply by acartro** [reply]
![image](https://user-images.githubusercontent.com/11137446/185076666-c4c19834-7619-4d44-8701-2c21e6cb6e20.png)
